cmd/labels.go

```diff
@@ -50,6 +51,7 @@ var labelsFile string
 
 func init() {
 	labelsCmd.Flags().StringVar(&labelsFile, "output", "labels.csv", "output file path")
+	checkURLsCmd.SilenceErrors = true
```

Comment:
Copy mistake error :)
The command names are different every time.

Reply:
This is one instance of easy to catch things if we have good test coverage

Reply:
Oh yes, that's my bad. 
And you're right good test coverage would solve the issue of manually checking if the tool works every time.